apps/new/widget/lib/Projects.jsx

```diff
@@ -0,0 +1,79 @@
+const fetchProjects = (props) => {
+  const app = props.app || "${alias_old}";
+  const type = props.type || "project";
+
+  const flattenObject = (obj, app, type) => {
+    const paths = [];
+    for (const key of Object.keys(obj)) {
+      const projects = Object.keys(obj?.[key]?.[app]?.[type] ?? {});
+      for (const project of projects) {
+        if (project && project.includes("_")) {
+          const convertedStr = project.replace(/_/g, "/");
+          paths.push(convertedStr);
+        }
+      }
+    }
+    return paths;
+  };
+
+  /** This may be moved to a separate, common SDK */
+  const fetchThings = () => {
+    const keys = Social.keys(`*/${app}/${type}/*`, "final", {
+      order: "desc",
+      subscribe: true,
+    });
```

Comment:
may be we should pass order and subscribe as parameter, to make it reusable?

Reply:
sure, we can do that!